### ISSUE TITLE
Admin Multiple Mail Sending

### ISSUE THREAD

**Issue by eka7a** [opencart/opencart#8051]
In e-mail sending page, when we select a specific customers and want to send an e-mail, if customers selected is than one page more, is sending mail to each customer as many as total pages
https://github.com/opencart/opencart/blob/f368a5f208564d46e6a0db7c292f339492a6f4f8/upload/admin/controller/marketing/contact.php#L136-L148

For example, if 20 customers were selected, since there are two pages, the same mail to a customer sent 2 times. I tested the master branch but it seems likely that there is this problem in all versions.

The same problem exists when to affiliates are selected.
https://github.com/opencart/opencart/blob/f368a5f208564d46e6a0db7c292f339492a6f4f8/upload/admin/controller/marketing/contact.php#L164-L176

**Comment by kenfai** [comment]
@eka7a Cannot replicate the issue you described.

Can you post a screenshot showing your Mail screen(route=marketing/contact) on how you setup your customers list to be sent?

**Comment by eka7a** [comment]
We did not test it on the live site. We tested it with test users on a cleanly installed site but we deleted the accounts we added as a test. However, if more customers are added in the screenshot below, there is a problem.

There is no code in codes here to disable the people who sent mail on previous page. The same value (`$this->request->post['customer']`) is submit on each page.
https://github.com/opencart/opencart/blob/f368a5f208564d46e6a0db7c292f339492a6f4f8/upload/admin/controller/marketing/contact.php#L136-L148

![test-mail](https://user-images.githubusercontent.com/5789807/87156387-44646f00-c2c5-11ea-9da5-637fd415116e.png)
